src/components/App.tsx

```diff
@@ -130,7 +130,7 @@ const App: React.FC<AppProps> = (props) => {
        * @see https://github.com/decentraland/explorer-website/pull/333#discussion_r1084094994
        * @see https://github.com/eordano/background-throttle
        */}
-      {!isElectron() && props.sound && <Audio track={`${process.env.PUBLIC_URL}/tone4.mp3`} play />}
+      {!isElectron() && props.sound && <Audio track={`${(import.meta as any).env.PUBLIC_URL}/tone4.mp3`} play />}
```

Comment:
If I'm not wrong, only environment variables that start with `VITE` are included into the meta env vars list, would you mind checking it out?